flexget/plugins/clients/qbittorrent.py

```diff
@@ -120,9 +124,61 @@ def connect(self, config):
         logger.debug('Successfully connected to qBittorrent')
         self.connected = True
 
-    def add_torrent_file(self, file_path, data, verify_cert):
+    def check_torrent_exists(self, file_path, verify_cert, hash_torrent):
         if not self.connected:
             raise plugin.PluginError('Not connected.')
+
+        if not isinstance(hash_torrent, str):
+            logger.error('Error getting torrent info, invalid hash {}', hash_torrent)
+            return False
+
+        hash_torrent = hash_torrent.lower()
+
+        url = f'{self.url}{self.api_url_info}'
+        params = {'hashes': hash_torrent}
+
+        try:
+            respose = self.session.request(
+                'get',
+                url,
+                params=params,
+                verify=verify_cert,
+            )
+        except RequestException as e:
+            logger.error('Error getting torrent info, request to hash {} failed', hash_torrent)
+            return False
+
+        if respose.status_code != 200:
+            logger.error(
+                'Error getting torrent info, hash {} search returned',
+                hash_torrent,
+                respose.status_code,
+            )
+            return False
+
+        try:
+            check_file = loads(respose.text)
+        except JSONDecodeError as e:
+            logger.error(
+                'Error getting torrent info for hash {}, response not a json',
+                hash_torrent,
+                file_path,
+            )
+            return False
+
+        if len(check_file) > 0:
```

Comment:
What sort of data does this endpoint return? Normally rather than doing a `len(check_file) >0` it would just be `if check_file`